observability/otel/otel-rn.adoc

```diff
@@ -12,6 +12,34 @@ You can use the {OTELName} xref:../../observability/otel/otel-forwarding-telemet
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="otel_3-5-1_{context}"]
+== Release notes for {OTELName} 3.5.1
+
+The {OTELName} 3.5.1 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} 0.119.0].
+
+[NOTE]
+====
+The {OTELName} 3.5.1 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.119.0.
+====
+
+[id="distr-tracing_3-5-1_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2025-22868[CVE-2025-22868]
+* link:https://access.redhat.com/security/cve/CVE-2025-27144[CVE-2025-27144]
+* link:https://access.redhat.com/security/cve/CVE-2025-29786[CVE-2025-29786]
+* link:https://access.redhat.com/security/cve/CVE-2025-30204[CVE-2025-30204]
+
+[id="otel_3-5-1_deprecated-functionality_{context}"]
+=== Deprecated functionality
+
+In the {OTELName} 3.5.1, the Loki Exporter, which is a temporary Technology Preview feature, remains deprecated. The Loki Exporter is planned to be removed in the {OTELName} 3.6. If you currently use the Loki Exporter for the OpenShift Logging 6.1 or later, replace the Loki Exporter with the OTLP HTTP Exporter.
```

Comment:
How gauranteed is the removal "If you currently use the Loki Exporter for the OpenShift Logging 6.1 or later"?. Would removing this sentence cause any issues?

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

Reply:
I amended the commit. It's removed, as discussed on Slack.